streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentTestUtils.java

```diff
@@ -77,13 +86,8 @@
     public static final TaskId TASK_2_3 = new TaskId(2, 3);
 
     public static final Set<TaskId> EMPTY_TASKS = emptySet();
-    public static final List<TaskId> EMPTY_TASK_LIST = emptyList();
-    public static final Map<TaskId, Long> EMPTY_TASK_OFFSET_SUMS = emptyMap();
     public static final Map<TopicPartition, Long> EMPTY_CHANGELOG_END_OFFSETS = new HashMap<>();
 
-    private AssignmentTestUtils() {}
```

Comment:
Why remove this? Do we need to instantiate this class now? (I only see static members still).

Reply:
> Do we need to instantiate this class now

No...isn't that exactly the reason we don't need this?

Reply:
Ah, note the `private`. The purpose of this constructor is to make it uninstantiable. I.e., it's "self-documenting" that it should only be a container for static members.

Reply:
Ah yeah ok. I'll put it back